mitiq/shadows/shadows_utils.py

```diff
@@ -101,52 +101,57 @@ def fidelity(
 
 def transform_to_cirq_paulistring(
     pauli_str: Union[str, mitiq.PauliString, cirq.PauliString[Any]]
-) -> Tuple[float, cirq.PauliString]:  # type: ignore
-    """Transforms mitiq.PauliString, cirq.PauliString, cirq.PauliSum or string to a cirq.PauliString class.
+) -> Tuple[float, cirq.PauliString[Any]]:
+    """Transforms mitiq.PauliString, cirq.PauliString, cirq.PauliSum or string
+    to a cirq.PauliString class.
 
     Args:
-      pauli_str: A mitiq.PauliString, cirq.PauliString, cirq.PauliSum or string.
+      pauli_str: A mitiq.PauliString, cirq.PauliString, cirq.PauliSum
+        or string.
 
     Returns:
       A tuple containing the coefficient and a cirq.PauliString.
     """
     if isinstance(pauli_str, cirq.PauliString):
         # Extract coefficient and return cirq.PauliString without coefficient
-        coeff = pauli_str.coefficient.real
-        new_pauli_str = cirq.PauliString(
+        coeff = np.real(pauli_str.coefficient)  # type: ignore
```

Comment:
Since you're in effect casting to a real value, I'd expect this `# type: ignore` is no longer needed.

Reply:
it will rise error when checking 
```
mitiq/shadows/shadows_utils.py:121: error: Argument 1 to "real" has incompatible type "Union[complex, float, Complex, Expr]"; expected "_SupportsReal[Any]"  [arg-type]
Found 1 error in 1 file (checked 78 source files)
```
The checking seems quite restrictive.  I need to keep this...

Reply:
don't worry, I have removed this function now! Thanks anyway, Misty!